test/jdk/java/awt/print/PrinterJob/PrintGlyphVectorTest.java

```diff
@@ -46,9 +69,7 @@ public void drawGVs(Graphics g) {
         FontRenderContext frc = g2d.getFontRenderContext();
         GlyphVector v = font.createGlyphVector(frc, testString);
 
-
-        float x = 50f,
-              y = 50f;
+        float x = 50f, y = 50f;
```

Comment:
A better way would be:
```java
float x = 50f;
float y = 50f;
```
Declaring the two variables on the same line is discouraged.